### PR TITLE
Functional tests - Refacto tests in 'BO > Shop parameters > Customer settings'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Refacto tests in ''BO > Shop parameters > Customer settings'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI and nightly are :green_heart: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/6814618227
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/pages/BO/shopParameters/customerSettings/titles/index.ts [hunk 1]
```diff
@@ -281,6 +324,76 @@ class Titles extends BOBasePage {
...
+    await this.waitForVisibleSelector(page, sortColumnDiv, 20000);
+  }
+
+  /* Pagination methods */
+  /**
+   * Get pagination label
+   * @param page {Page} Browser tab
+   * @return {Promise<string>}
+   */
+  getPaginationLabel(page: Page): Promise<string> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/shopParameters/customerSettings/titles/index.ts]:
```suggestion
  async getPaginationLabel(page: Page): Promise<string> {
```

File: tests/UI/pages/BO/shopParameters/customerSettings/groups/index.ts [hunk 2]
```diff
@@ -282,6 +328,103 @@ class Groups extends BOBasePage {
...
     return this.getAlertSuccessBlockParagraphContent(page);
   }
+
+  /* Pagination methods */
+  /**
+   * Get pagination label
+   * @param page {Page} Browser tab
+   * @return {Promise<string>}
+   */
+  getPaginationLabel(page: Page): Promise<string> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/shopParameters/customerSettings/groups/index.ts]:
```suggestion
  async getPaginationLabel(page: Page): Promise<string> {
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks

File: tests/UI/pages/BO/shopParameters/customerSettings/titles/index.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/shopParameters/customerSettings/titles/index.ts]: Done :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**